src/os/errors.go

```diff
@@ -0,0 +1,32 @@
+package os
+
+import (
+	"errors"
+)
+
+var (
+	ErrInvalid    = errors.New("invalid argument")
+	ErrPermission = errors.New("permission denied")
+	ErrClosed     = errors.New("file already closed")
+)
```

Comment:
These errors are really the same type of errors as the ones in file.go. I agree that it is a good idea to move them into a separate file, so can you move those errors also together with these errors?